.analysis_options

```diff
@@ -1,4 +1,6 @@
 analyzer:
+  language:
+    enableAssertInitializer: true
```

Comment:
That's indeed only for the test. I don't know how to enable this experimental feature only for the test and I think some changes in the analyzer package are needed to enable this experimental feature.

As it's an experimental feature this feature (check asserts in initializer list) could perhaps lives without test until its support improve (dartfmt, `LinterOptions` in analyzer...) wdyt ?